Source/NSKeyValueBinding.m

```diff
@@ -192,8 +192,12 @@ + (GSKeyValueBinding *) getBinding: (NSString *)binding
   if (!objectTable)
     return nil;
 
+  // NSLog(@"+++ called with %@, %@", binding, anObject);
```

Comment:
Do we want to leave this debug code in? If so, I would suggest to change it to `NSDebugLog` instead of commenting it out.

Reply:
Will do.